.changeset/smart-files-promise.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/components": patch
+---
+
+Removed "suspended" state from active-descendant focus zones.  Active descendant will now be active as soon as the `activeDescendantControl` element is focused, and cannot be deactivated by pressing `esc`.  This is a breaking change to `useFocusZone`, but this behavior is still considred to be in `alpha`.
```

Comment:
Is there a simpler way to explain this change? As someone not familiar with the intricacies of focus zones, this description is a little hard to follow

Reply:
Updated 😄

Reply:
Much clearer. Thank you ❤️